src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReferenceDiagnosticReport.java

```diff
@@ -0,0 +1,58 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example RFF+SRI:13/CH001137K/211010191093'
+ */
+@EqualsAndHashCode(callSuper = false)
+@Builder
+@Getter
+@AllArgsConstructor
+public class ReferenceDiagnosticReport extends Segment {
+
+    public static final String KEY = "RFF";
+    public static final String QUALIFIER = "SRI";
+    public static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
+
+    private final String referenceNumber;
+
+    public static ReferenceDiagnosticReport fromString(String edifactString) {
```

Comment:
Should have `final` before `String`

```suggestion
    public static ReferenceDiagnosticReport fromString(final String edifactString) {
```